raft/rafttest/network_test.go

```diff
@@ -52,7 +52,7 @@ func TestNetworkDelay(t *testing.T) {
 
 	w := time.Duration(float64(sent)*delayrate/2) * delay
 	// there are pretty overhead in the send call, since it genarete random numbers.
-	if total < w+10*delay {
+	if total < w {
```

Comment:
I think this is correct and the above comment should be removed. The `rand.Int63n` takes 63ns/op.
/cc @xiang90